experiments/Train.ipynb

```diff
@@ -304,7 +304,7 @@
     "    model_config.ingredient_min_count,\n",
     "    model_config.product_name_max_tokens,\n",
     "    model_config.product_name_max_length,\n",
-    "    load_dataframe(\"train\"),\n",
+    "    training_ds,\n",
```

Comment:
you can eventually remove `load_dataframe` from imports above :-)

Reply:
I thought I'd done it, but it was in the next PR :p